websites/S/South Park/dist/metadata.json

```diff
@@ -6,16 +6,15 @@
   },
   "url": [
     "www.southparkstudios.com",
-    "www.southpark.lat",
-    "southpark.cc.com"
```

Comment:
I believe these are used in some countries

Reply:
I don't even know why these were removed *cough* @EncryptedDev

Reply:
Oops, add `*.co.uk` as well then

Reply:
Actually I do: these all redirect to the first URL, just like the newly added one, so will remove that as well